Sources/Orbit/Components/Tabs.swift

```diff
@@ -0,0 +1,255 @@
+///         Tabs {
+///             Tab("one")
+///             Tab("two", style: .underlinedGradient(.ink))
+///             Tab("three", style: .underlined(.blueNormal))
+///         }
+///     }
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/structure/tabs/)
+public struct Tabs<Content: View>: View {
+
+    public let radius: CGFloat = 6
+    public let underlineHeight: CGFloat = 2
+
+    @Binding var selectedIndex: Int
+    let lineLimit: Int?
+    let distribution: TabsDistribution
+    let content: () -> Content
+
+    public var body: some View {
+        HStack(alignment: .center, spacing: 0) {
+            content()
+                .lineLimit(lineLimit)
+                .frame(maxWidth: maxTabWidth, minHeight: Layout.preferredSmallButtonHeight, maxHeight: .infinity)
+        }
+        .fixedSize(horizontal: false, vertical: true)
+        .opacity(0)
+        .backgroundPreferenceValue(Tab.PreferenceKey.self) { preferences in
+            GeometryReader { geometry in
+                tabs(for: preferences, in: geometry)
+            }
+        }
+        .background(background)
+    }
+
+    @ViewBuilder var background: some View {
+        RoundedRectangle(cornerRadius: radius)
+            .fill(Color.cloudLight)
+            .overlay(
+                RoundedRectangle(cornerRadius: radius)
+                    .stroke(Color.cloudNormal, lineWidth: BorderWidth.thin)
+            )
+    }
+
+    @ViewBuilder func tabs(for preferences: Tab.PreferenceKey.Value, in geometry: GeometryProxy) -> some View {
+        HStack(alignment: .center, spacing: 0) {
+            ForEach(preferences.indices, id: \.self) { index in
+                tab(
+                    index,
+                    lastIndex: preferences.indices.count - 1,
+                    preferences[index].label,
+                    style: preferences[index].style
+                )
+            }
+        }
+        .fixedSize(horizontal: false, vertical: true)
+        .background(activeTab(for: preferences, in: geometry), alignment: .leading)
+    }
+
+    @ViewBuilder func tab(_ index: Int, lastIndex: Int, _ label: String, style: TabStyle) -> some View {
+        Tab(label, style: style)
+            .foregroundColor(index == selectedIndex ? style.textColor : .inkNormal)
+            .lineLimit(lineLimit)
+            .frame(maxWidth: maxTabWidth, minHeight: Layout.preferredSmallButtonHeight, maxHeight: .infinity)
+            .overlay(separator(index: index, lastIndex: lastIndex), alignment: .trailing)
+            .contentShape(Rectangle())
+            .accessibility(addTraits: .isButton)
+            .onTapGesture {
+                withAnimation(.easeOut(duration: 0.25)) {
+                    selectedIndex = index
+                }
+            }
+    }
+
+    @ViewBuilder func separator(index: Int, lastIndex: Int) -> some View {
+        if 0 ..< lastIndex ~= index, [selectedIndex, selectedIndex - 1].contains(index) == false {
```

Comment:
This has two `contains`, but one is spelled using `~=`... wouldn't it be easier to read if both were spelled the same?